src/site/markdown/server/manage-webadmin.md

```diff
@@ -425,3 +425,72 @@ Note that several calls to this endpoint will be run in a sequential pattern.
 
 If the server restarts during the migration, the migration is silently aborted.
 
+## Creating address group
+
+You can use **webadmin** to define address groups.
+
+When a specific email is sent to the group mail address, every group member will receive it.
+
+This feature uses [Recipients rewrite table]() and requires the [RecipientRewriteTable mailet]() to be configured.
+
+### Listing groups
+
+```
+curl -XGET http://ip:port/address/groups
+```
+
+Will return the groups as a list of JSON Strings representing mail addresses. For instance:
+
+```
+["group1@domain.com", "group2@domain.com"]
+```
+
+Response codes:
+ - 200: Success
+ - 500: Internal error
+
+### Listing members of a group
+
+```
+curl -XGET http://ip:port/address/groups/group@domain.com
+```
+
+Will return the group members as a list of JSON Strings representing mail addresses. For instance:
+
+```
+["member1@domain.com", "member2@domain.com"]
+```
+
+Response codes:
+ - 200: Success
+ - 400: Group structure is not valid
+ - 403: Server does not own the requested domain
```

Comment:
It seems that it's not what swagger documentation is saying: 403 & 409 are not thrown, IMHO.

Reply:
have a look at RoutesGroups.addToGroup

Reply:
Yes, but in this documentation the `Response codes` seems to be for listing members of a group section.